src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.html

```diff
@@ -238,14 +238,17 @@ <h5>{{ 'order-details.shop-order-question' | translate }}</h5>
               placeholder="xxxxxxxxxx"
               [unmask]="false"
               [imask]="{ mask: ecoStoreMask }"
-              pattern="^[0-9]{10}"
+              [pattern]="additionalOrdersPattern"
               (click)="changeShopRadioBtn()"
               (ngModelChange)="ecoStoreValidation()"
             />
           </div>
           <button *ngIf="additionalOrders.controls.length < 5 && displayOrderBtn" class="addOrderBtn" (click)="addOrder()">
             {{ 'order-details.additional-orders' | translate }}
           </button>
+          <small class="text-danger" *ngIf="!displayOrderBtn && orderDetailsForm.get('additionalOrders').touched">
```

Comment:
i think it would be better to use getter for form control

Reply:
done